src/frontends/tensorflow/src/op/avg_pool.cpp

```diff
@@ -14,61 +14,47 @@ namespace tensorflow {
 namespace op {
 
 OutputVector translate_avg_pool_op(const NodeContext& node) {
-    Output<Node> ng_input = node.get_input(0);
+    auto op_type = node.get_op_type();
+    TENSORFLOW_OP_VALIDATION(node,
```

Comment:
I would recommend to move these 2 checks to helper function, e.g. 
default_op_checks() and apply for all translators

Reply:
Probably yes, we can think of it in the future. Now this is not one that we should spend time on it.

Reply:
Added.